Sources/Action.swift

```diff
@@ -18,12 +18,19 @@ import Result
 ///
 /// `Action` enforces serial execution, and disables the `Action` during the execution.
 public final class Action<Input, Output, Error: Swift.Error> {
-	private let deinitToken: Lifetime.Token
+	private enum Availability {
+		case enabledIdle
+		case enabledExecuting
+		case disabledIdle
+		case disabledExecuting
```

Comment:
Changed in https://github.com/ReactiveCocoa/ReactiveSwift/pull/400/commits/4acaf81a54c9da4940b8484cf745f73ae3fd14d7 anyway.